react/lib/components/Widget/Widget.tsx

```diff
@@ -535,7 +535,9 @@ export const Widget: React.FunctionComponent<WidgetProps> = props => {
       thisAmount = thisCurrencyObject ? thisCurrencyObject.float : undefined;
     }
     if (isValidCashAddress(address)) {
```

Comment:
"isValidCashAddress" is ambiguous... does this function check if it's a valid BCH address?

I think I'd prefer if we had a `isValidAddress` which accepted `address` and `currency` OR replace it with a function that takes in an address and returns the currency it matches (or false if no match). What happens to this section of code when we add a 3rd or 4th currency down the line?

Reply:
I believe it is named like this because of this : 
return xecaddr.isCashAddress(address);

I agree with you, will work on refactoring this part